core/pkg/opaprocessor/normalize_image_name.go

```diff
@@ -0,0 +1,13 @@
+package opaprocessor
+
+import (
+	"github.com/docker/distribution/reference"
+)
+
+func normalize_image_name(img string) (string, error) {
+	name, err := reference.ParseNormalizedNamed(img)
```

Comment:
We should consider calling [this](https://github.com/distribution/distribution/blob/main/reference/reference.go#L242C5-L242C5) function instead.

Reply:
Lokks like that function will return an [error](https://github.com/distribution/distribution/blob/807a836852c0f0297cd53027bef9d063c9af71d3/reference/reference.go#L247) for exactly the cases we are looking for, because ParseNormalizedNamed will return the image name with "docker.io" prefix